openpype/hosts/maya/api/plugin.py

```diff
@@ -238,6 +240,24 @@ def get_pre_create_attr_defs(self):
                     default=True)
         ]
 
+    def apply_settings(self, project_settings, system_settings):
```

Comment:
I didn't implement this by default because I wanted force developers to implement explicit settings apply. So any future developer can know what and which settings are applied, when and how, by looking into the file where the creator is, instead of magically applying them. That's my personal opinion how I think it should work, so don't take my word and let @antirotor and @BigRoy decide on this.

Reply:
Personally I think it'll lower the barrier of entry for contributions, but let's get some more opinions on this.

Reply:
I agree that it will lower the entry point barrier - for example in Maya case (but others too) it would mean explicitly adding tons of settings. But if we stick to the system, that most of the creator settings are in `project_settings["maya"]["create"]` and whatever else the creator needs will pull it explictitly, I think it is well-enough defined.

Reply:
@iLLiCiTiT you happy to continue with this then?

Reply:
From my point of view yes. I would be good to clearly state soemwhere (in docstrings?) how the logic works and where to define the settings, but overall it is responsibility of the creator developer to document it properly inside the plugin itself

Reply:
> @iLLiCiTiT you happy to continue with this then?

@antirotor has spoken :) not a blocker for me.